build/Build.cs

```diff
@@ -114,7 +114,10 @@ ITargetDefinition TestDefinition(ITargetDefinition targetDefinition, Target depe
                     .EnableBlameCrash()
                     .SetBlameCrashDumpType("full")
                     .EnableBlameHang()
-                    .SetBlameHangTimeout(TimeSpan.FromMinutes(20).TotalMilliseconds.ToString())
+                    // This is set high since when a hang dump is collected it is saved into /tmp/
+                    // On windows the dump collecting utility appears to be missing and so nothing is collected.
+                    // Setting high means we will have time to get in and collect a dump manually
+                    .SetBlameHangTimeout(TimeSpan.FromMinutes(9999).TotalMilliseconds.ToString())
```

Comment:
I don't know if we want to tie up build agents for `6.94375` days when something hangs.

I get we need to spot it's hanging (can team city alert us?) and then get someone to access the machine and take a process dump so we need it to remain online for a while. Could we reduce this down a bit so that if for some reason we don't spot a hung build, we at least release the agent after a reasonable period of time.

Reply:
Yeh, dial it down a bit and maybe a bit extra to the comment so the reader knows there is another timeout that kicks in before this one.

Reply:
Ahh auto merge!! Lol, never mind!!

Reply:
https://github.com/OctopusDeploy/Halibut/pull/415